__tests__/app.test.js

```diff
@@ -70,14 +68,80 @@ describe('2. GET /api/articles/:article_id', () => {
 				});
 			});
 	});
-	test('returns a 404 error when the article is not found', () => {
+	test('returns a 404 error when the article ID is not found', () => {
 		return request(app)
 			.get('/api/articles/999')
 			.expect(404)
 			.then(({ body }) => {
 				expect(body).toEqual({ msg: 'Invalid ID' });
```

Comment:
You are using the 'invalid ID' message for this 404 error and the 400 as well below. Please change one of them to differentiate and use the appropriate message.